esp/esp/survey/templatetags/survey.py

```diff
@@ -47,6 +49,14 @@
 
 register = template.Library()
 
+@register.filter
+def midValue(sizeLs0):
+    sizeLst = int(sizeLs0)
+    if sizeLst%2 == 1:
+        return math.ceil((sizeLst / 2))
+    else:
+        return sizeLst/2+0.1
```

Comment:
I think you're just worried about getting a float here, so `sizeLst/2.` would be better.